rai/show.go

```diff
@@ -127,3 +127,14 @@ func (tx *TransactionResult) Show() {
 		}
 	}
 }
+
+func (tx *TransactionAsyncResult) Show() {
+	for i, r := range tx.Results {
+		if i > 0 {
+			fmt.Println()
+		}
+
+		fmt.Println(r.RelationID)
+		fmt.Println(r.Table...)
+	}
+}
```

Comment:
👀 wow, i can't believe how simple that was!!! 🤔 Is this because `Table` somehow already implements some kind of printable interface?

Reply:
`Table` is a simple go array of interfaces so the `...` did the job :smile:

Reply:
ohhh i see! Very nifty :)